dev-support/Jenkinsfile_GitHub

```diff
@@ -470,6 +471,144 @@ pipeline {
                         }
                     }
                 }
+                stage ('yetus jdk17 hadoop3 checks') {
```

Comment:
Here, I think we can just apply the changes of this file to master and branch-3, and not apply to other branches for now. So we do not need to add branch condition in the when section.